projects/cds/README.md

```diff
@@ -103,4 +103,60 @@ In order for CDS events do be sent, a consent with the ID Profile must be define
 ```
 INSERT_UPDATE ConsentTemplate;id[unique=true];name[lang=en];description[lang=$lang];version[unique=true];baseSite(uid)[unique=true,default=exampleUid];exposed
 ;PROFILE;"Allow SAP Commerce Cloud, Context-Driven Services tracking";"We would like to store your browsing behaviour so that our website can dynamically present you with a personalised browsing experience and our customer support agents can provide you with contextual customer support.";1;;true
-```
\ No newline at end of file
+```
+
+## CDS shell app
+
+### The Library Builds
+
+Run the following command to ensure the library builds
+
+```
+yarn build:core:lib:cds
+```
+
+### The Shell Starts
+
+Run the following command to ensure the shell starts 
+
+```
+yarn start:cds
+```
+
+### Unit Tests are Passing
+
+The unit tests need to be passing.
+
+Run the following commands to perform unit tests:
+
+```
+yarn test:cds
+```
+
+When you run these commands, Chrome opens, and you can see the progress of the tests, with detailed information, including whether the tests pass.
+
+### End-To-End Tests is Passing
+
+Need to ensure that this feature is foolproof. 
+
+Run the following command to perform end-to-end tests: 
+
+- Manually
+```
+yarn start:cds
+yarn e2e:cy:cds:run:vendor
+```
+
+Note: Make sure you have your spartacus instance running before running the e2e command, which will run the e2e test in headless mode.
+
+- Automatically
+```
+yarn e2e:cy:cds:start-open
+```
+
+Note: A spartacus instance does not need to be running as it will start one for you from production mode, and you would need to choose the `merchandising-carousel.e2e-spec.ts` to view the test running in cypress user interface.
+
+## Other commands for cds
+
+There exist other commands for cds, but you would need to take a look at `package.json` and find a grouping of scripts which has `:cds` as part of its command.
```

Comment:
Maybe phrase it like this: _all other CDS commands can be found in `package.json` and have `:cds` as part of their name_.

Reply:
short and sweet :)